EvidenceApi/V1/Factories/ResponseFactory.cs

```diff
@@ -38,10 +38,11 @@ public static ResidentResponse ToResponse(this Resident domain, string? referenc
             this DocumentSubmission domain,
             DocumentType documentType,
             DocumentType? staffSelectedDocumentType = null,
-            Document? document = null
+            Document? document = null,
+            Claim? claim = null
```

Comment:
`Claim` contains `Document` so you pass `Claim` instead of both of them?

Reply:
👍